### ISSUE TITLE
Custom colors

### ISSUE THREAD

**Issue by vertigo220** [tom-james-watson/reddit-comment-collapser#22]
Add ability to adjust colors, mainly to make them brighter against dark backgrounds

**Comment by tom-james-watson** [comment]
Do you mean to change the colors globally?

**Comment by vertigo220** [comment]
Yeah. The preset colors aren't always easy to see, so the ability to customize them would allow users to change them to make them more visible. Ideally, there would be different color choices for light and dark backgrounds. Another possibility would be an option to adjust the width, which would also help make them more visible, and might be easier to add and require less tweaking.
